src/NuGet.Core/NuGet.Packaging/Signing/Native/CmsFactory.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Now that the `Native` folder contains both managed and native implementations, the folder name is outdated.  Rename the `Native` folder to `Cms`.

Reply:
Thanks! Renamed.